alumnium/alumni.py

```diff
@@ -23,11 +23,15 @@
 
 
 class Alumni:
-    def __init__(self, driver: Page | WebDriver, model: Model = None):
+    def __init__(self, driver: Page | WebDriver, model: Model = None, mobile_platform: str = "Android"):
```

Comment:
I don't think we need it. We should instead attempt to check what Appium driver is passed via `driver` (iOS or Android). I believe there are some properties we can check that are available in `driver` variable...